workers/common/src/main/kotlin/common/env/MavenSettingsGenerator.kt

```diff
@@ -37,9 +37,11 @@ class MavenSettingsGenerator : EnvironmentConfigGenerator<MavenDefinition> {
         /** The name of the file that is generated by this generator. */
         private const val TARGET_NAME = ".m2/settings.xml"
 
-        /** The schema location for Maven's settings.xml. */
-        private const val SCHEMA = "\"http://maven.apache.org/SETTINGS/1.1.0 " +
-                "http://maven.apache.org/xsd/settings-1.1.0.xsd\""
+        /** The attributes of the `settings` root element. */
+        private const val ROOT_ATTRIBUTES = "xmlns=\"http://maven.apache.org/SETTINGS/1.0.0\" " +
+                "xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" " +
+                " xsi:schemaLocation=\"http://maven.apache.org/SETTINGS/1.0.0 " +
+                "https://maven.apache.org/xsd/settings-1.0.0.xsd\""
```

Comment:
This changes the expected settings version to 1.0.0.
[Here](https://maven.apache.org/ref/3.2.1/maven-settings/settings.html) is an example with the version 1.1.0.